packages/react-player/cypress/component/autoplay.cy.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2023 Design Barn Inc.
+ */
+
+import { PlayerState } from 'common';
+import React, { useState } from 'react';
+
+import { Controls } from '../../src/controls';
+import { DotLottiePlayer } from '../../src/react-player';
+import { PlayerStateWrapper } from '../support/player-state-wrapper';
+
+describe('Autoplay', () => {
+  it('should not play when `autoplay` = `false`', () => {
```

Comment:
@afsal-ali Could you rename this test or the one below, as they share the same name but dont check the same thing, also if theres the autoplay prop, shouldnt it be playing? You also check that the state is playing, so maybe this title is wrong ?